### DISCUSSION TITLE
Deezer setup

### DISCUSSION THREAD

**Discussion by thedinz** [RandomNinjaAtk/arr-scripts#146] [Q&A] [answered]
I just signed up fo deezer premium and switched from tidal to deezer in my config but is there no setup? I dont even need a deezer premium account.

It appears to be downloading from deezer using freyr? without needing any of my account information or tokens?

**Comment by RandomNinjaAtk** [comment, marked as answer]
Put the token in the config….

**Comment by thedinz** [comment]
Geesh, it must be wild dealing with us idiots.

Yeah its different from tidal, I didn’t know the ARL: area was for Deezer token, I did try the wiki but didn’t see any instruction on that, my apologies..

Is this information somewhere that I just missed?


------ Original Message ------
From "RandomNinjaAtk" ***@***.******@***.***>>
To "RandomNinjaAtk/arr-scripts" ***@***.******@***.***>>
Cc "thedinz" ***@***.******@***.***>>; "Author" ***@***.******@***.***>>
Date 1/1/24, 9:45:34 AM
Subject Re: [RandomNinjaAtk/arr-scripts] Deezer setup (Discussion #146)


Put the token in the config….

**Reply by RandomNinjaAtk** [reply]
Most of the information is still in the old repos... Wiki is open for edits, so feel free to contribute...
